index.html

```diff
@@ -3447,6 +3447,19 @@ <h3>Syntactic Sugar</h3>
         </section>
 
       </section>
+
+      <section>
+        <h3>Mappings for Verifiable Credentials</h3>
+
+        <p>
+The standard representation of a Verifiable Credential is defined in this specification (which uses the media type <code>application/vc+ld+json</code>).
+Other representations can be used, provided that a mapping is defined between that representation and the standard one.
+This mapping definition should state whether it is unidirectional or bidirectional.
+Deployments that choose to do so can map the representation to the standard one and process it accordingly.
+A media type should be defined for each Verifiable Credential representation so that it can be used to distinguish between representations.
```

Comment:
There is also "should" language that either needs to use RFC "SHOULD" or be changed to "is expected to".

We've had reviewers get confused between lowercase should vs. uppercase SHOULD (since they have very different normative meanings) and have kept the spec using RFC language only when we mean it in the upper-case sense.

Reply:
```suggestion
A media type is expected to be defined for each Verifiable Credential representation, which can then be used to distinguish between representations.
```